com.github.fries1234.ncsa-mosaic.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.github.fries1234.ncsa-mosaic</id>
+    <name>NCSA Mosaic</name>
+    <summary>The NSCA Mosaic Web Browser</summary>
+    <metadata_license>CC-BY-SA-3.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/fries1234/ncsa-mosaic/blob/master/COPYRIGHT</project_license>
+    <description>
+        <p>The NCSA Mosaic Web Browser is a browser that came out in 1993. It is one of the first widely available web browsers and the one of the browsers that made the web explode in popularity.</p>
+    </description>
+    <url type="homepage">https://github.com/fries1234/ncsa-mosaic</url>
+    <url type="bugtracker">https://github.com/fries1234/ncsa-mosaic/issues</url>
+    <screenshots>
+        <screenshot type="default">
+            <caption>A NCSA Mosaic browser window displaying the first Web Page.</caption>
+            <image>https://github.com/flathub/com.github.fries1234.ncsa-mosaic/raw/master/mosaic.png</image>
```

Comment:
the problem here is that when buillding it won't find the screenshot as the repo doesn't exist.

the best solution is to store them upstream.